### PR TITLE
Update utility app - cache tab

### PR DESCRIPTION
**Pull request by unpluggedk**
#109

### CODE REVIEW

File: MSAL/test/app/ios/MSALTestAppCacheViewController.m [hunk 1]
```diff
@@ -234,8 +228,8 @@ - (void)viewDidLoad {
     }];
     
     _tokenRowActions = @[ deleteTokenAction, expireTokenAction ];
-    _rtRowActions = @[ tombstoneTokenAction, invalidateAction ];
-    _clientIdRowActions = @[ deleteAllAction ];
+    _rtRowActions = @[ deleteTokenAction, invalidateAction ];
+    _environmentRowActions = @[ deleteAllAction ];
     
     [[NSNotificationCenter defaultCenter] addObserverForName:MSALTestAppCacheChangeNotification
```
Comment (oldalton) [inline MSAL/test/app/ios/MSALTestAppCacheViewController.m]: Not a change in this pull request, but I think here we should call removeObserver somewhere as well.

File: MSAL/test/app/ios/MSALTestAppCacheViewController.m [hunk 2]
```diff
@@ -254,82 +248,93 @@ - (void)didReceiveMemoryWarning {
...
+    
+    if ([item isKindOfClass:[MSALAccessTokenCacheItem class]])
+    {
+        environment = [NSURL URLWithString:((MSALAccessTokenCacheItem *)item).authority].msalHostWithPort;
+    }
+    else if ([item isKindOfClass:[MSALRefreshTokenCacheItem class]])
     {
-        clientIdTokens = [NSMutableArray new];
-        [userTokens setObject:clientIdTokens forKey:clientId];
+        environment = ((MSALRefreshTokenCacheItem *)item).environment;
```
Comment (oldalton) [inline MSAL/test/app/ios/MSALTestAppCacheViewController.m]: Can we just use item.user.environment here instead for both MSALAccessTokenCacheItem and MSALRefreshTokenCacheItem?

Comment (unpluggedk) [inline MSAL/test/app/ios/MSALTestAppCacheViewController.m]: 👍

File: MSAL/test/app/ios/MSALTestAppCacheViewController.m (hunk 1 above)
Comment (unpluggedk) [inline MSAL/test/app/ios/MSALTestAppCacheViewController.m]: Yeah, i agree it's a best practice. But in here, it is one of the viewcontroller for the tabbar that will not be unloaded for all intended purposes.

**Pull request merged**